templates/rhdm78-authoring-ha.yaml

```diff
@@ -869,8 +869,6 @@ objects:
           - name: KIE_MBEANS
             value: "${KIE_MBEANS}"
 ## OpenShift Enhancement BEGIN
-          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED
```

Comment:
@desmax74 
We can not remove this environment variable, rather need to change the value from 'false' to 'true'. The reason behind it is that by turning that switch on it will allow Business Contral console to be able to discovery KIE Server instances with 'OpenShiftStartupStrategy'. So those two env variables must be in pairs as the following:

"KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED" = "true" <=> "KIE_SERVER_STARTUP_STRATEGY" = "OpenShiftStartupStrategy"
"KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED" = "false" <=> "KIE_SERVER_STARTUP_STRATEGY" = "ControllerBasedStartupStrategy"

Reply:
@rhtevan  Yep is present at line https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/343/files#diff-1eb85ffee474c725d66b9148d46c18a0R1126